src/components/DaySectionGrid.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Webtoon } from "../types/webtoon.ts";
+import WebtoonCard from "./WebtoonCard.tsx";
+
+function DaySectionGrid({ webtoons }: { webtoons: Webtoon[] }) {
+  return (
+    <ul className={` 
+      flex flex-wrap gap-2  
+      pb-[20px]
+      `}
+    >
+      {webtoons.map((webtoon: Webtoon) => (
+        <li key={webtoon.id} 
+          className={`items-center py-[10px] w-[168px] border-2`}>
+          <WebtoonCard 
+          id={webtoon.id}
+          title={webtoon.title}
+          thumbnail_url={webtoon.thumbnail_url}
+          average_rating={webtoon.average_rating}
+          />
```

Comment:
```suggestion
          <WebtoonCard 
            id={webtoon.id}
            title={webtoon.title}
            thumbnail_url={webtoon.thumbnail_url}
            average_rating={webtoon.average_rating}
          />
```
@electrohyun  공백이 포함되지 않은 것 같습니다. 프리티어 적용이 혹시 안되있을까요?

Reply:
@ssi02014 안녕하세요! 제가 몸살을 겪어 답변이 늦었던 점 죄송합니다. 리뷰 감사합니다!! 프로젝트가 시작되고 나서 Prettier라는 것이 있는 걸 알게 되었는데요, 수정하면 상관없는 코드나 파일들이 전부 Changed 되어 리뷰어분들께 혼란을 드릴 것 같아 사용하지 않고 있었습니다...😭 실례가 안된다면 이번 PR에 대한 리뷰 반영한 뒤 프리티어 style 커밋을 추가하는 것이 좋을지, 아니면 별도 브랜치를 파서 진행하는 것이 좋을지 생각을 여쭤봐도 될까요?

Reply:
별도의 PR 생성해서 진행하시면 좋을 것 같아요!

Reply:
넵!! 의견 주셔서 감사합니다! 바로 진행해볼게요!